apps/spruce/src/components/Header/AuxiliaryDropdown.tsx

```diff
@@ -49,6 +68,8 @@ export const AuxiliaryDropdown: React.FC<AuxiliaryDropdownProps> = ({
       to: getProjectSettingsRoute(projectIdentifier),
       onClick: () => sendEvent({ name: "Clicked project settings link" }),
     },
+    // We shouldn't show any inverse links if the waterfall page hasn't been released.
+    ...(showWaterfallPage ? [inverseLink] : []),
```

Comment:
I _think_ instead of this condition we probably want to use the result of `useAdminBetaFeatures`? That will allow flipping the setting to change this instead of having to remove this value from code.